iFixit/react-commerce
# Add footer partners

closes #143 and #176 

## Changelog

- Add seed data for footer partner links
- Improve footer partner code
- Replace placeholder with no image fixie illustration

## QA

1. Visit the Vercel PR preview
2. Visit parts product list page
3. Verify that footer has partners logos
4. Verify that products without images display the image shown [here](https://github.com/iFixit/react-commerce/issues/176)
